pkg/kubectl/cmd/create_job_test.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	batchv1 "k8s.io/api/batch/v1"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fake "k8s.io/client-go/kubernetes/fake"
+	clienttesting "k8s.io/client-go/testing"
+	cmdtesting "k8s.io/kubernetes/pkg/kubectl/cmd/testing"
+)
+
+var submittedJob *batchv1.Job
+
+func TestCreateJobFromCronJob(t *testing.T) {
+	testNamespaceName := "test"
+	testCronJobName := "test-cronjob"
+	testImageName := "fake"
+
+	expectedLabels := make(map[string]string)
+	expectedAnnotations := make(map[string]string)
+	expectedLabels["test-label"] = "test-value"
+	expectedAnnotations["cronjob.kubernetes.io/instantiate"] = "manual"
```

Comment:
might want to define a const for "cronjob.kubernetes.io/instantiate" because I see these popping up quite a few times in the code.

Reply:
I'm not sure I'm wanna keep it, since it doesn't do us anything.